src/modules/missionHelper/main.ts

```diff
@@ -8,6 +8,15 @@ export default (async ({ LSSM, $m, $mc, getSetting, setSetting }) => {
         document.querySelector('.missionNotFound')
     )
         return;
+    getSetting('hide_on_Krankentransport').then(hideBool => {
+        const missionID = parseInt(LSSM.$utils.getMissionTypeInMissionWindow());
+
+        if (
+            Object.values(LSSM.$t('transfer_missions')).includes(missionID) &&
+            hideBool
+        )
+            return false;
+    });
```

Comment:
this does not work as `return` onyl exits the promise but not the modules function.

I would suggest the following version (untested):
```suggestion
    const doNotLoad = await getSetting('hide_on_Krankentransport').then(hideBool => {
        const missionID = parseInt(LSSM.$utils.getMissionTypeInMissionWindow());

        return
            Object.values(LSSM.$t('transfer_missions')).includes(missionID) &&
            hideBool;
    });
    if (doNotLoad) return;
```